src/App.test.js

```diff
@@ -2,8 +2,6 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+test('renders app', () => {
+  render(<App />);
```

Comment:
should we maybe leave the old code commented out as a quick example reference in the future when we do want to add asserts?

Reply:
Sure we can keep it commented in - in general rendering the whole app is not worthy a unit test / integration test anyway. Do you want to have e2e tests later on?

Otherwise I'd maybe even remove this testcase and move it to singular components later on

Reply:
Rendering the entire app can be useful as a smoke test.